### PR TITLE
Log entity ids before querying in TDR plugin (#2265)

### PR DESCRIPTION
**Pull request by noah-aviel-dove**
Author (before primary review)

- [x] PR title references issue
- [x] Title of main commit references issue
- [x] PR is linked to Zenhub issue
- [x] Added `HCA` label <sub>or this PR does not target an `hca/*` branch</sub>
- [x] Created issue to track porting these changes <sub>or this PR does not need porting</sub> 
- [x] Added `[r]` prefix at start of commit title <sub>or this PR does not require reindexing</sub>
- [x] Added `reindex` label <sub>or this PR does not require reindexing</sub>
- [x] Freebies are blocked on this PR <sub>or there are no freebies in this PR</sub>
- [x] Freebies are referenced in commit titles <sub>or there are no freebies in this PR</sub>
- [x] This PR is blocked by previous PR in chain <sub>or this PR is not chained to another PR</sub>
- [x] Added `chain` label to the blocking PR <sub>or this PR is not chained to another PR</sub>
- [x] Ran `make requirements_update` <sub>or this PR does not change requirements*.txt</sub>
- [x] Mentioned `make requirements` in UPGRADING.rst <sub>or this PR does not change requirements*.txt</sub>
- [x] Documented upgrade of personal deployments in UPGRADING.rst <sub>or this PR does not require upgrading</sub>

Primary reviewer (before pushing merge commit)

- [x] Checked `reindex` label and `[r]` prefix of commit title
- [x] Rebased and squashed branch
- [x] Sanity-checked history
- [x] Build passes in `sandbox` <sub>or commented that sandbox will be skipped</sub>
- [x] Reindexed `sandbox` <sub>or this PR does not require reindexing</sub>
- [x] Added PR reference to merge commit

Primary reviewer (after pushing merge commit)

- [ ] Moved linked issue to Merged
- [ ] Pushed merge commit to Github
- [ ] Moved freebies to Merged column <sub>or there are no freebies in this PR</sub> 
- [ ] Shortened chain <sub>or this PR is not the base of another PR</sub>
- [ ] Verified that `N reviews` labelling is accurate
- [ ] Commented on demo expectations <sub>or labeled as `no demo`</sub>
- [ ] Pushed merge commit to Gitlab
- [ ] Reindexed `dev` <sub>or this PR does not require reindexing</sub>
- [ ] Deleted PR branch from Github and Gitlab
- [ ] Unassign reviewer from PR

### CODE REVIEW

**Review by achave11 — APPROVED** [review]
LGTM!

File: src/azul/plugins/repository/tdr/__init__.py [hunk 1]
```diff
@@ -204,6 +204,7 @@ def retrieve_rows(entity_type: str, entity_ids: Set[str]):
             non_pk_columns = bundle.data_columns if entity_type.endswith('_file') else bundle.metadata_columns
             columns = ', '.join(non_pk_columns | {pk_column})
             uuid_in_list = ' OR '.join(f'{pk_column} = "{entity_id}"' for entity_id in entity_ids)
+            log.info('Retrieving %s %s entities...', len(entity_ids), entity_type)
```
Comment (hannes-ucsc) [inline src/azul/plugins/repository/tdr/__init__.py:207]:
```suggestion
            log.info('Retrieving %i entities of type %r ...', len(entity_ids), entity_type)
```

similarly for the log message at the conclusion of this method.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Comment by noah-aviel-dove** [comment]
We could also use [azul.strings.pluralize](https://github.com/DataBiosphere/azul/blob/c99bd3d66559712e2926e6c5853587c03b86c5b1/src/azul/strings.py#L25) here if we want to be strict about grammar. But my feeling is that this isn't importan when the text doesn't end getting displayed in the service.

**Review by hannes-ucsc — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
I agree.

**Comment by hannes-ucsc** [comment]
For demo show log message in CloudWatch for `dev` deployment.

**Pull request merged**